bfvmm/src/vmcs/src/vmcs_intel_x64_check_controls.cpp

```diff
@@ -361,12 +384,19 @@ vmcs_intel_x64::check_control_enable_ept_checks()
 void
 vmcs_intel_x64::check_control_enable_pml_checks()
 {
-    if (!is_enabled_pml())
+    using namespace vmcs::primary_processor_based_vm_execution_controls;
+    using namespace vmcs::secondary_processor_based_vm_execution_controls;
+
+    bool secondary_ctls = activate_secondary_controls::is_enabled();
+    bool pml = enable_pml::is_enabled();
+    bool ept = enable_ept::is_enabled();
+
+    if (!pml || !secondary_ctls)
```

Comment:
I would drop it where it makes sense. If you end up with a line that is huge, obviously keep it, but I like having the full name there as I don't have to figure out which namespace it came from.